Tests/secrets_white_list.json

```diff
@@ -1189,7 +1189,8 @@
       "walla.com",
       "www.securityadvisor.io",
       "fp.tools",
-      "www.securityadvisor.io"
+      "www.securityadvisor.io",
+      "https://integration.cybereason.net"
```

Comment:
```suggestion
      "integration.cybereason.net"
```

Reply:
@Itay4  Did that at the beginning, kept yelling for secrets until I added the https

Reply:
need to understand why, we shouldnt add the protocol here
@jochman can you have a look please?

Reply:
we should not do it after a problem we encountered with excluding domains (we can by accident push email in a@<secret-excluded-domain>.)
But, this change is fine, it's unlikely that there's someone under <name>@integration.cybereason.net.